scripts/qa_crowbarsetup.sh

```diff
@@ -4455,6 +4455,27 @@ function onadmin_reapply_openstack_proposals
     done
 }
 
+function is_precheck_passed
+{
+    precheck_output=$1
+    required=false
+    for line in $precheck_output; do
+        if $required; then
```

Comment:
The `[[` is not necessary because this line runs the command `/bin/false` which produces a non-zero exit code. I personally like this form, it's a lot more like using an actual language :smile: .